libs/sdk-react-native/ios/RNBreezSDK.swift

```diff
@@ -181,11 +181,11 @@ class RNBreezSDK: RCTEventEmitter {
         }
     }
     
-    @objc(receivePayment:description:resolver:rejecter:)
-    func receivePayment(_ amountSats:UInt64, description:String, resolver resolve: @escaping RCTPromiseResolveBlock, rejecter reject: @escaping RCTPromiseRejectBlock) -> Void {
+    @objc(receivePayment:resolver:rejecter:)
+    func receivePayment(_ reqData:[String: Any], resolver resolve: @escaping RCTPromiseResolveBlock, rejecter reject: @escaping RCTPromiseRejectBlock) -> Void {
         do {
-            let lnInvoice = try getBreezServices().receivePayment(amountSats: amountSats, description: description)
-            resolve(BreezSDKMapper.dictionaryOf(lnInvoice: lnInvoice))        
+            let response = try getBreezServices().receivePayment(reqData: reqData)
+            resolve(BreezSDKMapper.dictionaryOf(receivePaymentResponse: response))
         } catch let err {
             rejectErr(err: err, reject: reject)
         }
```

Comment:
The reqData needs to be converted into a ReceivePaymentRequest type
```
        if let receivePaymentRequest = BreezSDKMapper.asReceivePaymentRequest(reqData: reqData) {
            do {
                let response = try getBreezServices().receivePayment(reqData: receivePaymentRequest)                
                resolve(BreezSDKMapper.dictionaryOf(receivePaymentResponse: response))
            } catch let err {
                reject(RNBreezSDK.TAG, "Error calling receivePayment \(err)", err)
            }
        } else {
            rejectErr(err: SdkError.Generic(message:"Invalid reqData"), reject: reject)            
        }
```

Reply:
Done as part of 4efe618